workflow/rules/ref.smk

```diff
@@ -0,0 +1,13 @@
+rule get_genome:
```

Comment:
Isn't this supposed to be called get_transcriptome?

Reply:
... and the output `"resources/transcriptome.fasta"`?

Reply:
Oups. Copy-pasta...